data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
 				`	"I'm honored. Thank you."`
 					decline
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
+			`	"There might be," he says. "Check the spaceports. If anyone has a job to be done, they will approach you. And there are almost always a few Korath ships that seem bent on destruction instead of loot, and those need to be dealt with."`
+				decline
+
+
+mission "Remnant: Heavy Laser"
+	name "Retrieve Heavy Lasers for the Remnant"
+	description "An engineer on <planet>, a Remnant world, has offered to pay you one million credits in exchange for obtaining two Heavy Lasers."
+	minor
+	source
+		government Remnant
+	to offer
+		has "license: Remnant"
+		random < 50
+	on offer
+		require "Heavy Laser"
+		conversation
+			`As you are browsing the spacesport shops, a woman approaches you and introduces herself as a researcher at one of their weapons facilities. She pauses before beginning to recite a quick tune: "We can see that your ship has weapons that match human techniques, but are unlike anything we have record of. What kind of weapons are they?"`
```

Comment:
Changed to add that to "awkwardly" like the other missions in this trio.

Also added mission restrictions so that the other two missions in this trio will not be offered if one of them has been completed.

Reply:
Adjusted her dialogue to refer to beam weapons. Is that clear enough, or should I simplify it to simply laser cannons and make the mission agnostic as to whether the player brings back regular or heavy lasers?